packages/blade/src/components/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import type { MenuProps } from './';
+import { Menu, MenuDivider, MenuItem, MenuOverlay, MenuHeader, MenuFooter } from './';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import { CopyIcon, TestIcon, TicketIcon, UserIcon } from '~components/Icons';
+import { Avatar } from '~components/Avatar';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import { Text } from '~components/Typography';
+import { Link } from '~components/Link';
+
+export default {
+  title: 'Components/Menu',
+  component: Menu,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+    trigger: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+} as Meta<any>;
```

Comment:
Pass prop to Meta

Reply:
will fix it in docs PR where  I wrote documentation